benchmark_runner/common/ocp_resources/odf/template/04_local_volume_set_template.yaml

```diff
@@ -19,7 +19,7 @@ spec:
     deviceTypes:
       - disk
       - part   # <-- Remove this if not using partitions
-    #deviceMechanicalProperties:
+    deviceMechanicalProperties:
     #  - NonRotational   # <-- For HDD change to Rotational
     #minSize: 0Ti   # <-- Uncomment and modify to limit the minimum size of disk used
-    #maxSize: 0Ti   # <-- Uncomment and modify to limit the maximum size of disk used
+    maxSize: 1250Ti   # <-- Uncomment and modify to limit the maximum size of disk used
```

Comment:
This isn't a very clean way to distinguish between different types of disks.

Reply:
Right, I will modify it to filter by vendor and model:
    models:
      - PERC H730P Adp
    vendors:
      - DELL

Reply:
That's not ideal either, as it assumes specifics about our setup (what happens when somebody runs it on a very different cluster?).  But it's better.  Can't you inspect /dev/disk/... to determine what type of disk it is (NVMe vs. SATA)?

Reply:
The only option is - NonRotational but its same for NVMe and SSD.
So the only option is by size or by vendor.
Here is the full yaml template for LocalVolumeSet:
```
apiVersion: local.storage.openshift.io/v1alpha1
kind: LocalVolumeSet
metadata:
  name: example-autodetect
spec:
  nodeSelector:
    nodeSelectorTerms:
      - matchExpressions:
          - key: kubernetes.io/hostname
            operator: In
            values:
              - worker-0
              - worker-1
  storageClassName: example-storageclass 
  volumeMode: Filesystem
  fsType: ext4
  maxDeviceCount: 10
  deviceInclusionSpec:
    deviceTypes: 
      - disk
      - part
    deviceMechanicalProperties:
      - NonRotational
    minSize: 10G
    maxSize: 100G
    models:
      - SAMSUNG
      - Crucial_CT525MX3
    vendors:
      - ATA
      - ST2000LM
```
For more details: https://docs.openshift.com/container-platform/4.14/storage/persistent_storage/persistent_storage_local/persistent-storage-local.html

@jeniferh, Do you think its safe enough to choose model and vendor for disks auto detect ?

Reply:
I think the filters are going to be very environmentally specific no matter what, including maxDeviceCount, although I suppose you can inspect the disks dynamically to grab the filter values. I've never actually used models or vendors, from the code it just says it needs to match what 'lsblk' provides. The other option instead of Discovery mode w/ LocalVolumeSet Filters is to just use LocalVolume and list each devicePath